quiz.css

```diff
@@ -0,0 +1,58 @@
+* {
+	font-family: sans-serif;
+}
+
+.product-card {
+	display: inline-block;
+	width: 30%;
+	margin: 0.5rem;
+	border: 1px solid lightblue;
+}
+
+.product-title {
+	border: 1px dotted lightgray;
+	text-align: center;
+}
+
+.product-title h1 {
+	margin-top: 1rem;
+	margin-bottom: 1rem;
+}
+
+.product-info * {
+	display: block;
+}
+
+.product-info img {
+	margin: 0.5rem auto;
+	width: 75%;
+}
+
+.product-description {
+	text-align: justify;
+}
+
+.availability {
+	display: block;
```

Comment:
I only found the `class="availability"` on `div`s which are already `display: block`